mteb/benchmarks/benchmarks.py

```diff
@@ -438,6 +438,55 @@
     contacts=["gowitheflow-1998"],
 )
 
+MTEB_ARA = Benchmark(
+    name="MTEB(ara, v1)",
+    tasks=MTEBTasks(
+        get_tasks(
+            languages=["ara"],
+            tasks=[
+                # Classification
+                "AJGT",
+                "HotelReviewSentimentClassification",
+                "LanguageClassification",
+                "MassiveIntentClassification",
+                "MultiHateClassification",
+                "MultilingualSentimentClassification",
+                "OnlineStoreReviewSentimentClassification",
+                "RestaurantReviewSentimentClassification",
+                "TweetEmotionClassification",
+                "TweetSarcasmClassification",
+                "TweetSentimentClassification",
+                "MassiveScenarioClassification",
+                # Pair Classification
+                "ArEntail",
+                "XNLI",
+                # BitextMining
+                "IWSLT2017BitextMining",
+                # Reranking
+                "MIRACLReranking",
+                "NamaaMrTydiReranking",
+                # Retrieval
+                "MIRACLRetrieval",
```

Comment:
You both have this with and without hard negatives. I would probably remove this one